.travis.yml

```diff
@@ -32,13 +24,21 @@ before_install:
 go_import_path: github.com/cfpb/rhobot
 
 install:
-    - go get github.com/Sirupsen/logrus
-    - go get github.com/urfave/cli
-    - go get -u github.com/davecgh/go-spew/spew
-    - go get github.com/lib/pq
-    - go get gopkg.in/yaml.v2
-    - go get github.com/flosch/pongo2
-    - go get gopkg.in/gomail.v2
+    - go get github.com/tools/godep
+    - godep restore
+    # - go get github.com/Sirupsen/logrus
```

Comment:
Can we just remove these commented out lines?

Reply:
yup